lib/medium-admin.php

```diff
@@ -136,6 +136,12 @@ public static function add_meta_boxes_post($post) {
   public static function save_post($post_id, $post) {
     if (defined('DOING_AUTOSAVE') && DOING_AUTOSAVE) return;
 
+    $allowed_post_types = (array) apply_filters("medium_allowed_post_types", array("post"));
```

Comment:
I'm a fan of encouraging people return the types they should. I know PHP is dynamic and anything goes, but it's no hardship to require an array. So yes please :)